ftplugin/vader.vim

```diff
@@ -49,3 +58,8 @@ augroup vader_syntax
   " autocmd FileType <buffer> call vader#syntax#include(1, '$')
 augroup END
 
+let b:undo_ftplugin = 'setl sw< ts< sts< et< cms< isk<'
+      \ . ' | exe "au! vader_syntax * <buffer>"'
+      \ . ' | unlet b:vader_label b:vader_eos | ' . get(b:, 'undo_ftplugin', '')
```

Comment:
I'm not sure of the purpose of the last part ` | ' . get(b:, 'undo_ftplugin', '')`. Do we need it? The example given in `:help undo_ftplugin` does not have it.